### PR TITLE
Updating react version

### PR DESCRIPTION
**Pull request by pmarrone**
This PR updates react to version 16.8 

Also, fixing some linter issues

### CODE REVIEW

File: admin/package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
     "dev": "./node_modules/.bin/webpack-dev-server --mode development",
     "build": "./node_modules/.bin/webpack",
     "test": "jest",
-    "eslint": "eslint src"
+    "eslint": "eslint srr --ext .js --ext .jsx"
```
Comment (HalfVoxel) [inline admin/package.json]: This looks like a typo. Shouldn't it still be 'src' and not 'srr'?

**Comment by andersroos** [comment]
Great job, everything seems to work in the GUI. I think all lint errors/deprecation warnings should be fixed before merging since there were no errors/warnings on master.

File: admin/package.json (hunk 1 above)
Comment (pmarrone) [reply admin/package.json]: This is a typo alright

File: admin/src/Components/Login.jsx [hunk 2]
```diff
@@ -31,14 +31,14 @@ export default class Login extends React.Component
...
-                                <input ref="username" className="uk-form-large uk-form-width-large" type="text" placeholder="Användarnamn" />
+                                <input ref={c => { this.username = c; }} className="uk-form-large uk-form-width-large" type="text" placeholder="Användarnamn" />
                             </div>
                         </div>
 
                         <div className="uk-form-row">
                             <div className="uk-form-icon">
                                 <i className="uk-icon-lock"/>
-                                <input ref="password" className="uk-form-large uk-form-width-large" type="password" placeholder="Lösenord" />
+                                <input ref={c => { this.password = c; }} className="uk-form-large uk-form-width-large" type="password" placeholder="Lösenord" />
```
Comment (pmarrone) [inline admin/src/Components/Login.jsx]: It's probably a good idea to drop the ref thing altogether and just use a controlled component (by adding state somewhere. Also, hooks are now available)

Comment (andersroos) [reply admin/src/Components/Login.jsx]: Yes, I agree.

File: admin/package.json (hunk 1 above)
Comment (pmarrone) [reply admin/package.json]: Should be fixed now! Also eslint issues should have been addressed.

**Pull request merged**